lib/diagnostic.rb

```diff
@@ -106,10 +106,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = { age: 24, first_name: 'Lauren' }
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = Hash.new { |hash| hash[:first_name] << " hates Trump." }
```

Comment:
Trying to create a hash with the way that this is currently written will cause a stack overflow.  `person.default = "#{person[:first_name]} hates Trump."`.